app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -33,10 +33,12 @@
   }
 }
 
-@include govuk-grid-row($class: "gem-c-feedback__grid-row");
+.gem-c-feedback__grid-row {
+  @extend %govuk-grid-row;
+}
 
 .gem-c-feedback__column-two-thirds {
-  @include govuk-grid-column(two-thirds, $class: false)
+  @include govuk-grid-column(two-thirds)
```

Comment:
Nitpick: no semicolon at the end of this line (although presumably it passed the linter?).

Reply:
This code was removed in a following commit.